srml/staking/src/lib.rs

```diff
@@ -17,51 +17,57 @@
 //! # Staking Module
 //! <!-- Original author of paragraph: @gavofyork -->
 //!
-//! The staking module is the means by which a set of network maintainers (known as _authorities_ in some contexts and _validators_ in others)
-//! are chosen based upon those who voluntarily place funds under deposit. Under deposit, those funds are rewarded under
-//! normal operation but are held at pain of _slash_ (expropriation) should the staked maintainer be found not to be
-//! discharging their duties properly.
+//! The staking module is the means by which a set of network maintainers (known as _authorities_ in some contexts
+//! and _validators_ in others) are chosen based upon those who voluntarily place funds under deposit. Under deposit,
+//! those funds are rewarded under normal operation but are held at pain of _slash_ (expropriation) should the
+//! staked maintainer be found not to be discharging its duties properly.
 //!
-//! You can start using the Staking module by implementing the staking [`Trait`].
+//! You can start using the staking module by implementing the staking [`Trait`](./trait.Trait.html).
```

Comment:
It's a name so it deserves capitalisation.
```suggestion
//! You can start using the Staking module by implementing the `staking::` [`Trait`](./trait.Trait.html).
```